docs/source/jit.rst

```diff
@@ -694,8 +694,56 @@ Disable JIT for Debugging
     function.
 
 
+Inspecting Code
+^^^^^^^^^^^^^^^
+
+    TorchScript provides a code pretty-printer for all ScriptModule instances. This
```

Comment:
I'm not sure to what extent in the future if we'll have Torchscript without Scriptmodules (e.g. classes, etc). Maybe rewrite more generally or call section `Inspecting ScriptModuled` instead?